examples/ai_safety_v2/safe_shutdown/safe_shutdown.py

```
@@ -0,0 +1,454 @@
+        
+        try:
+            func()
+            phase_duration = time.time() - phase_start
+            self.phase_times[name] = phase_duration
+            logger.info(f"✓ {name} complete ({phase_duration:.2f}s)")
+            
+        except Exception as e:
+            phase_duration = time.time() - phase_start
+            self.phase_times[name] = phase_duration
+            error_msg = f"{name} failed: {e}"
+            logger.error(f"✗ {error_msg}")
```

Ruff (0.14.10)

Use `logging.exception` instead of `logging.error`

Replace with `exception`

(TRY400)